uniffi_bindgen/src/bindings/python/templates/ObjectTemplate.py

```diff
@@ -1,13 +1,13 @@
 {%- let obj = ci.get_object_definition(name).unwrap() %}
 
 class {{ type_name }}(object):
-    {%- match obj.primary_constructor() %}
-    {%- when Some with (cons) %}
+{%- match obj.primary_constructor() %}
```

Comment:
I was having real trouble reading the askama control flow, so I re-indented all control flow in this file, which I think makes it far more readable, but lmk if you think I've made things worse here.